
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.3.0, mkdocs-material-8.2.8">
    
    
      
        <title>RawTools - Proteomics QC</title>
      
    
    
      <link rel="stylesheet" href="../assets/stylesheets/main.644de097.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.e6a45f82.min.css">
        
      
    
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
    <script>__md_scope=new URL("..",location),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      
  


  
  


  <script>function gtag(){dataLayer.push(arguments)}window.dataLayer=window.dataLayer||[],gtag("js",new Date),gtag("config","G-9FFCYK4MWP"),document.addEventListener("DOMContentLoaded",function(){document.forms.search&&document.forms.search.query.addEventListener("blur",function(){this.value&&gtag("event","search",{search_term:this.value})}),"undefined"!=typeof location$&&location$.subscribe(function(e){gtag("config","G-9FFCYK4MWP",{page_path:e.pathname})})})</script>
  <script async src="https://www.googletagmanager.com/gtag/js?id=G-9FFCYK4MWP"></script>


    
    
  </head>
  
  
    
    
    
    
    
    <body dir="ltr" data-md-color-scheme="" data-md-color-primary="none" data-md-color-accent="none">
  
    
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#rawtools" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Proteomics QC" class="md-header__button md-logo" aria-label="Proteomics QC" data-md-component="logo">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54z"/></svg>

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Proteomics QC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              RawTools
            
          </span>
        </div>
      </div>
    </div>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5z"/></svg>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" aria-label="Clear" tabindex="-1">
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


<nav class="md-nav md-nav--primary" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Proteomics QC" class="md-nav__button md-logo" aria-label="Proteomics QC" data-md-component="logo">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54z"/></svg>

    </a>
    Proteomics QC
  </label>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
      

  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        Home
      </a>
    </li>
  

    
      
      
      

  
  
  
    <li class="md-nav__item">
      <a href="../installation/" class="md-nav__link">
        Installation
      </a>
    </li>
  

    
      
      
      

  
  
  
    <li class="md-nav__item">
      <a href="../server/" class="md-nav__link">
        Setup an online server
      </a>
    </li>
  

    
      
      
      

  
  
  
    <li class="md-nav__item">
      <a href="../how-to-access-the-admin-panel/" class="md-nav__link">
        Admin panel
      </a>
    </li>
  

    
      
      
      

  
  
  
    <li class="md-nav__item">
      <a href="../how-to-add-a-project/" class="md-nav__link">
        Create a project
      </a>
    </li>
  

    
      
      
      

  
  
  
    <li class="md-nav__item">
      <a href="../how-to-add-a-pipeline/" class="md-nav__link">
        Create a pipeline
      </a>
    </li>
  

    
      
      
      

  
  
  
    <li class="md-nav__item">
      <a href="../how-to-submit-raw-files/" class="md-nav__link">
        Submit RAW files
      </a>
    </li>
  

    
      
      
      

  
  
  
    <li class="md-nav__item">
      <a href="../datalake/" class="md-nav__link">
        Datalake
      </a>
    </li>
  

    
      
      
      

  
  
    
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" data-md-toggle="toc" type="checkbox" id="__toc">
      
      
        
      
      
        <label class="md-nav__link md-nav__link--active" for="__toc">
          RawTools
          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <a href="./" class="md-nav__link md-nav__link--active">
        RawTools
      </a>
      
        

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#parameters" class="md-nav__link">
    Parameters
  </a>
  
</li>
      
    </ul>
  
</nav>
      
    </li>
  

    
      
      
      

  
  
  
    <li class="md-nav__item">
      <a href="../maxquant/" class="md-nav__link">
        MaxQuant
      </a>
    </li>
  

    
      
      
      

  
  
  
    <li class="md-nav__item">
      <a href="../api/" class="md-nav__link">
        API
      </a>
    </li>
  

    
      
      
      

  
  
  
    <li class="md-nav__item">
      <a href="../about/" class="md-nav__link">
        About
      </a>
    </li>
  

    
      
      
      

  
  
  
    <li class="md-nav__item">
      <a href="../issues/" class="md-nav__link">
        Help/Issues
      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#parameters" class="md-nav__link">
    Parameters
  </a>
  
</li>
      
    </ul>
  
</nav>
                  </div>
                </div>
              </div>
            
          
          <div class="md-content" data-md-component="content">
            <article class="md-content__inner md-typeset">
              
                


<h1 id="rawtools">RawTools</h1>
<blockquote>
<p>Kovalchik KA, Colborne S, Spencer SE, Sorensen PH, Chen DDY, Morin GB, et al. RawTools: Rapid and Dynamic Interrogation of Orbitrap Data Files for Mass Spectrometer System Management. J Proteome Res. 2018. doi:10.1021/acs.jproteome.8b00721</p>
</blockquote>
<p>RawTools is a comprehensive and user-friendly software designed for the analysis of mass spectrometry-based proteomics data. It is an open-source tool that provides a range of features for processing and analyzing raw proteomics data, including quality control checks, filtering, normalization, peak detection, and quantification.</p>
<p>The tool supports various file formats commonly used in proteomics, including RAW, MGF, and mzML files, and offers a range of visualization options to aid in data exploration and interpretation. The software also includes advanced features such as label-free quantification, statistical analysis, and data normalization, making it an ideal tool for proteomics researchers.</p>
<p>One of the key strengths of RawTools is its user-friendly interface, which allows researchers to easily navigate through their data and perform complex analyses without the need for extensive programming skills. The software is regularly updated, and its development is driven by a community of researchers and bioinformaticians, ensuring that it stays up-to-date with the latest advancements in proteomics research.</p>
<p>Overall, RawTools is a powerful and versatile tool that can help researchers to efficiently process, analyze, and interpret their proteomics data, providing valuable insights into the biological systems under investigation.</p>
<p>More information is available in the Github <a href="https://github.com/kevinkovalchik/RawTools">repository</a>.</p>
<h2 id="parameters">Parameters</h2>
<p>Here is the help text that explains the different parameters of <em>RawTools</em> that are relevant for <em>Proteomics QC</em>. However, not all of them were tested.</p>
<pre><code class="language-raw">========== PARAMETERS HELP ==========

Below you will find information on all the parameters available to this application. 
The Parameter field is a (hopefully) descriptive name of the parameter which might 
be referenced when encountering an error. Short form and Long form are how you invoke
the parameters on the command line. You can use either form. 

Required indicates whether the application requires the parameter have a value in order to run.

INFO         DESCRIPTION                                                          ---------------

Parameter: RawFiles                 Indicates input file(s) to be processed, separated by a space if
Short form: -f                      there are multiple files. Must be Thermo .raw files. You must use 
Long form: -files                   either -f or -d to indicate the file(s) to process. 
Required: False

Parameter: Parse                    Parses raw file meta and scan data and writes the output to a 
Short form: -p                      tab-delimited text file. Typically either this output or the 
Long form: -parse                   quant output (-q) is used unless your aim is to simply create 
                                    an MGF or to observe broad metrics using -x.
Long form: -parse
Required: False 

Parameter: Quant                    Similar to parse (-p), but also quantifies reporter ions and write 
Short form: -q                      results to output matrix. Use of this flag requires you also specify 
Long form: -quant                   the reagents used for isobaric labeling with the -r argument 
Required: False                     (e.g. -r TMT10)

Parameter: LabelingReagent          Required for reporter ion quantification. Reagents used to label
Short form: -r                      peptides, required if using quant option. Available options are: 
Long form: -labellingreagent        {TMT0, TMT2, TMT6, TMT10, TMT11, TMT16, iTRAQ4, iTRAQ8}.
Required: False 

Parameter: UnlabeledQuant           Calculate areas of precursor peaks and writes them to the parse or 
Short form: -u                      quant file (ParentPeakArea column). This option is to be used in 
Long form: -unlabeledquant          combination with -p or -q.
Required: False 

Parameter: WriteMGF                 Writes a standard MGF file. To specify a mass cutoff use the -c 
Short form: -m                      argument.
Long form: -mgf
Required: False 

Parameter: MgfMassCutoff            Specify a mass cutoff to be applied when generating MGF files. 
Short form: -c                      May be of use if removal of reporter ions is desired prior to 
Long form: -masscutoff              searching of MS2 spectra. Default is 0.
Required: False 

Parameter: OutputDirectory          The directory in which to write output. Can be a relative or 
Short form: -o                      absolute path to the directory. If it is a relative path it 
Long form: -out                     will be placed inside the directory containing the respective 
Required: False                     raw file. Note that relative paths should not start with a slash. 
                                    If this is left blank the directory where the raw file is stored 
                                    will be used by default.

Parameter: Metrics                  Write a txt file containing general metrics about the MS run.
Short form: -x  
Long form: -metrics
Required: False 

Parameter: Chromatogram             Write a chromatogram to disk. Should be in the format 
Short form: -chro                   &quot;-chro [order][type]&quot;, where order is the MS order (or a combination
Long form: -chromatograms           of orders) and type is T, B, or TB (TIC, base peak and both, 
Required: False                     respectively). For example, to generate MS1 and MS2 TIC and base 
                                    peak chromatograms, invoke &quot;-chro 12TB&quot;. Or, to generate a MS2 TIC, 
                                    invoke &quot;-chro 2T&quot;.

Parameter: RefineMassCharge         Refine precursor charge and monoisotopic mass assignments. 
Short form: -R                      Highly recommended if monoisotopic precursor selection was 
Long form: -refinemasscharge        turned off in the instrument method (or peptide match on a 
Required: False                     QE instrument).

Parameter: MinCharge                The minimum charge to consider when refining precursor 
Short form: -min                    mass and charge.
Long form: -mincharge
Required: False 

Parameter: MaxCharge                The maximum charge to consider when refining precursor 
Short form: -max                    mass and charge.
Long form: -maxcharge
Required: False 

Parameter: FastaDB                  Required for an X! Tandem search during QC. Path to a 
Short form: -db                     fasta protein database.
Long form: -fastadb
Required: False 

Parameter: FixedModifications       Fixed modifications to pass to the search, if desired. 
Short form: -fmods                  Use mass@aminoacid1,mass@aminoacid2 format. It is important 
Long form: -fixedmods               that the values are separated with a comma and not spaces. 
Required: False                     Invoke &quot;&gt;RawTools -modifications&quot; to see examples of some 
                                    common modifications         

Parameter: VariableModifications    Variable modifications to pass to the search, if desired. 
Short form: -vmods                  Use mass@aminoacid1,mass@aminoacid2 format. It is important 
Long form: -variablemods            that the values are separated with a comma and not spaces. 
Required: False                     Invoke &quot;&gt;RawTools -modifications&quot; to see examples of some 
                                    common modifications     

Parameter: XTandemDirectory         Specify the path to the X! Tandem directory (the directory 
Short form: -X                      containing &quot;tandem.exe&quot;) if you want to run a database search 
Long form: -xtandem                 as part of QC.
Required: False 

Parameter: NumberSpectra            The number of MS2 spectra to be passes to the search engine as 
Short form: -N                      an MGF file. Defaults to 10,000. If N is greater than the number
Long form: -numberspectra           of MS2 scans in a raw file, all MS2 scans will be used.
Required: False 

Parameter: LogDump                  Write the instrument logs from all provided raw files to disk.
Short form: -l  
Long form: -logdump
Required: False 
</code></pre>

              
            </article>
          </div>
        </div>
        
      </main>
      
        <footer class="md-footer">
  
    <nav class="md-footer__inner md-grid" aria-label="Footer">
      
        
        <a href="../datalake/" class="md-footer__link md-footer__link--prev" aria-label="Previous: Datalake" rel="prev">
          <div class="md-footer__button md-icon">
            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12z"/></svg>
          </div>
          <div class="md-footer__title">
            <div class="md-ellipsis">
              <span class="md-footer__direction">
                Previous
              </span>
              Datalake
            </div>
          </div>
        </a>
      
      
        
        <a href="../maxquant/" class="md-footer__link md-footer__link--next" aria-label="Next: MaxQuant" rel="next">
          <div class="md-footer__title">
            <div class="md-ellipsis">
              <span class="md-footer__direction">
                Next
              </span>
              MaxQuant
            </div>
          </div>
          <div class="md-footer__button md-icon">
            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M4 11v2h12l-5.5 5.5 1.42 1.42L19.84 12l-7.92-7.92L10.5 5.5 16 11H4z"/></svg>
          </div>
        </a>
      
    </nav>
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    <script id="__config" type="application/json">{"base": "..", "features": [], "search": "../assets/javascripts/workers/search.5e67fbfe.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.config.lang": "en", "search.config.pipeline": "trimmer, stopWordFilter", "search.config.separator": "[\\s\\-]+", "search.placeholder": "Search", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version.title": "Select version"}}</script>
    
    
      <script src="../assets/javascripts/bundle.c44cc438.min.js"></script>
      
    
  </body>
</html>